packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleApp.vcxproj

```diff
@@ -167,33 +167,7 @@
   <ItemGroup>
     <Natvis Include="$(ReactNativeWindowsDir)\Folly\Folly.natvis" />
   </ItemGroup>
-  <!-- We need to reference Microsoft.ReactNative to build, but we can't because SampleLibraryCPP already references it, and CppWinRT
```

Comment:
><!-- We ne [](start = 2, length = 10)

I am just curious: was this code the reason why I had all the winmd merge issues? :)